lib/lightspeed/item.rb

```diff
@@ -58,5 +58,10 @@ class Item < Lightspeed::Resource
     def prices
       @prices ||= Lightspeed::Prices.new(self.Prices)
     end
+
+    def default_vendor
```

Comment:
yes. it builds a method `default_vendor` that finds and returns, a vendor with id defaultVendorID, or returns a previously found or included one. because the API is pretty regular we can do things like this. (slow, but regular)